tests/functional_tests/transfer.py

```
@@ -61,18 +77,18 @@ def reset(self):
           'velvet lymph giddy number token physics poetry unquoted nibs useful sabotage limits benches lifestyle eden nitrogen anvil fewest avoid batch vials washing fences goat unquoted',
           'peeled mixture ionic radar utopia puddle buying illness nuns gadget river spout cavernous bounced paradise drunk looking cottage jump tequila melting went winter adjust spout',
           'dilute gutter certain antics pamphlet macro enjoy left slid guarded bogeys upload nineteen bomb jubilee enhanced irritate turnip eggs swung jukebox loudly reduce sedan slid',
         ]
-        self.wallet = [None] * len(seeds)
-        for i in range(len(seeds)):
+        self.wallet = [None] * len(self.seeds)
+        for i in range(len(self.seeds)):
             self.wallet[i] = Wallet(idx = i)
             # close the wallet if any, will throw if none is loaded
             try: self.wallet[i].close_wallet()
             except: pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(BaseException): ...`

(SIM105)